flow/record/adapter/csvfile.py

```diff
@@ -52,3 +54,28 @@ def close(self):
         if self.fp and not is_stdout(self.fp):
             self.fp.close()
         self.fp = None
+
+
+class CsvfileReader(AbstractReader):
+    fp = None
+
+    def __init__(self, path, selector=None, **kwargs):
+        self.selector = make_selector(selector)
+        if path in (None, "", "-"):
+            self.fp = sys.stdout
+        else:
+            self.fp = open(path, "r", newline="")
+        self.reader = csv.reader(self.fp)
+
+    def close(self):
+        if self.fp:
+            self.fp.close()
+        self.fp = None
+
+    def __iter__(self):
+        row = next(self.reader)
+        desc = RecordDescriptor("csv/reader", [("string", col) for col in row])
```

Comment:
Maybe nice feature to support headerless CSV files and specify the columns through an argument of the adapter?

Reply:
hmm yes, was thinking this as well. Will look into it.

Reply:
Adding it to the adapter should be no problem, would you also expect this to be easy to do via `rdump` as well, for example via a new CLI argument, or would specifying a `csvfile://stuff.csv?fields=a,b,c` be sufficient?

Reply:
Yeah I meant the latter method, supplying through the adapter query arguments should be sufficient.